va_explorer/va_data_management/views.py

```diff
@@ -23,11 +23,17 @@ def get_queryset(self):
         # Restrict to VAs this user can access and prefetch related for performance
         queryset = self.request.user.verbal_autopsies().prefetch_related("location", "causes", "coding_issues").order_by("id")
 
+        # TODO: For now, we are not displaying the filters for the Field Worker on the VA index page,
+        # since many do not apply to them. This prevents data passed through the params from being
+        # passed to the VAFilter
+        if self.request.user.is_fieldworker():
+            self.filterset = VAFilter(data=None, queryset=queryset)
```

Comment:
You do an early return here and just do `return queryset` instead since it doesn't need all the stuff below it for filterset.

But it doesn't matter a whole lot and is kinda nit-picky.

Reply:
@timster Thank you! I tried a couple of combinations of things here, but couldn't get it to work correctly without setting the filterset in this way. Do you think this is okay as is? I might be missing something totally obviously here.